pcp/versions/next/unauthorized.md

```diff
@@ -0,0 +1,50 @@
+Unauthorized
+===
+
+When a broker declines to deliver a message because it fails authorization,
+it must send an *unauthorized* message back to the original message sender.
+
+A clinet may send a message to multiple recipients while it's only authorized
+to send that message to some of them:
+
+```
+    client A                    broker B                multiple clients
+       |                           |                           |
+       |         1 message         |                           |
+       |-------------------------->| 2                         |
+       |                           |                           |
+       |      3 unauthorized       |                           |
+       |<--------------------------|                           |
+       |                           |                           |
+       |                           |         4 message         |
+       |                           |-------------------------->|
+       |                           |                           |
+```
+
+Client A sends a message to possibly multiple clients through broker B (1).
+B processes (2) the message, checks its authorization, which fails for at
+least some of the clients A addressed the message to. As a result B sends
```

Comment:
"for at least" should be removed; you should describe the example without uncertainty